solutions/swb-reference/integration-tests/tests/isolated/projects/get.test.ts

```diff
@@ -58,4 +60,12 @@ describe('Get Project negative tests', () => {
       }
     });
   });
+
+  test('unauthorized user gets 401', async () => {
+    try {
+      await anonymousSession.resources.projects.project(project2Id).get();
+    } catch (e) {
+      checkHttpError(e, new HttpError(401, {}));
+    }
```

Comment:
Anonymous user throws 401 here but a 403 for create project. Maybe a post-release item, but error codes should be consistent for the same test case..?

Reply:
This is due to non GET requests requiring CSRF token and cookieParser/csurf handling the error vs. when GETs don't need the CSRF token, this terminates in the authentication middleware and becomes a 401. This is something that we want to address post launch.